gnd/src/main/java/com/google/android/gnd/rx/Result.java

```diff
@@ -62,12 +63,32 @@ public Throwable getError() {
     return error;
   }
 
-  public static <T, R> Function<T, Single<Result<R>>> wrapErrors(
+  public static <T, R> Function<T, Single<Result<R>>> wrapSingle(
       @NonNull Function<T, Single<R>> fn) {
     return (T value) -> fn.apply(value).map(Result::success).onErrorReturn(Result::error);
   }
 
-  public static <T> Consumer<? super Result<T>> unwrapErrors(
+  public static <T, R> Function<T, Observable<Result<R>>> wrapObservable(
+      @NonNull Function<T, Observable<R>> fn) {
+    return (T value) -> fn.apply(value).map(Result::success).onErrorReturn(Result::error);
+  }
+
+  /**
+   * Wraps observables in a Result without applying any operation. Equivalent to mapping "id" on the
```

Comment:
What does mapping "id" mean here?

Reply:
I just mean to say that no additional transformation is performed on the object aside from moving it into a result. I'll rewrite this so that it's clearer. 

I used the term id because "wrapping" something like we do here is equivalent to calling `map(x -> x)`. The x -> x part is usually called the identity function, or just id. Java takes care of that part for us implicitly though because of the way Result is defined, so it's kinda silly to mention it!